distributed/scheduler.py

```diff
@@ -3736,6 +3741,9 @@ async def feed(
                 if teardown:
                     teardown(self, state)
 
+    def log_worker_event(self, worker=None, topic=None, msg=None):
+        self.log_event(topic, msg)
```

Comment:
Open question, should we include the worker address in a list of topics?

Reply:
I don't have strong thoughts on this. This could, for example, be useful to get all the events logged on specific workers. Though I'm inclined to hold off on that for the time being just to keep things simple initially